crates/paimon/Cargo.toml

```diff
@@ -34,6 +34,7 @@ storage-memory = ["opendal/services-memory"]
 storage-fs = ["opendal/services-fs"]
 
 [dependencies]
+rand = "0.8.5"
```

Comment:
Should this be `dev-dependencies`?

Reply:
You're right, that was an oversight on my part.